app/controllers/users_controller.rb

```diff
@@ -27,8 +28,9 @@ def update_authenticated_user
   end
```

Comment:
Would be good to document in `create` what the `prepend` is doing. I can see that getting confusing. Consider seeing how others document these methods when doing a `prepend`.